index.html

```diff
@@ -3,14 +3,46 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    
-    <title>My first Microverse student project</title>
-
+    <title>Luis's portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
     <link rel="stylesheet" href="css/style.css">
   </head>
-  <body>
 
-    <h1>Hello Microverse!</h1>
+  <body>
+    <header class="header">
+      <nav class="navbar d-flex">
+        <div class="nav-logo">My Logo</div>
+        <div class="hamburger"></div>
```

Comment:
#### Optional suggestion
- [ ] `My logo` should be a link not a text, kindly use a tag to wrap the logo.